File: python/bc1071.py
fVal = int(input())
sVal = int(input())
soma = 0

if fVal > sVal:
    aux = fVal
    fVal = sVal
    sVal = aux

for i in range(fVal+1, sVal):
    if i % 2 != 0:
        soma += i

print(soma)

File: cpp/bc1071.cpp
#include <iostream>

using namespace std;

int main()
{
	int fVal, sVal, sum = 0;
	
	cin >> fVal >> sVal;
	
	if(fVal > sVal)
	{
		int aux = fVal;
		fVal = sVal;
		sVal = aux;
	}
	
	for(int i = fVal+1; i < sVal; i++)
		if(i % 2 != 0) sum += i;
	
	
	cout << sum << endl;
}
